src/main/java/mate/academy/service/ConnectionUtil.java

```diff
@@ -0,0 +1,29 @@
+package mate.academy.service;
+
+import java.sql.Connection;
+import java.sql.DriverManager;
+import java.sql.SQLException;
+import java.util.Properties;
+
+public class ConnectionUtil {
+    private static final String DB_URL = "jdbc:mysql://127.0.0.1:3306/test?user=root";
+    private static final Properties DB_PROPERTIES;
+    private static final String USERNAME = "root";
+    private static final String PASSWORD = "Nero0410";
```

Comment:
Storing the password in the source code is a security risk. Consider using environment variables or a configuration file that is not included in the version control system.